components/grid/selection/rows.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Row Selection
+page_title: Grid - Rows Selection
+description: Learn how to select row in Blazor Grid component. Explore the selected rows. Discover row selection bevahior when combined with other Grid features. Try the practical sample code for row selection.
+slug: components/grid/selection/rows
+tags: telerik,blazor,grid,selection,rows
+previous_url: /components/grid/selection/single,/components/grid/selection/multiple
+position: 3
+---
+
+# Row Selection
+
+The Grid component offers support for single or multiple row selection. You can select a row with mouse click and through checkbox column. You can access the collection of selected rows, use this collection and manipulate it. You can follow and respond to the event of selection.
+
+In this article:
+
+* [Rows Selection Options](#rows-selection-options)
+	* [Click-Only Selection](#click-selection)
+	* [Checkbox Selection](#checkbox-selection)
```

Comment:
Merge these three sections into one with a single example. We already have a checkbox column example in the Columns section.